scripts/generate-sitemap.js

```diff
@@ -3,23 +3,67 @@
 const fs = require('fs')
 const { parseStringPromise, Builder } = require('xml2js')
 
+const weekMs = 7 * 24 * 60 * 60 * 1000
+const nowISO = new Date().toISOString()
+
+const getQuery = (chain) => `
+  query {
+    topTokens(pageSize: 100, page: 1, chain: ${chain}, orderBy: VOLUME) {
+      address
+    }
+  }
+`
+const chains = ['ETHEREUM', 'ARBITRUM', 'OPTIMISM', 'POLYGON', 'BASE', 'BNB', 'CELO']
```

Comment:
can we import this from src? I think we already have an array of supported chain names and this will be easy to forget to update

Reply:
no, this is a generated gQL type and i didn't find such an array already existing

Reply:
okay so you didn't need to manually add these, these were auto-generated? we may want to not check these in then similar to how we don't check in the other GQL schema

Reply:
This also seems identical to `BACKEND_SUPPORTED_CHAINS` in `src/graphql/data/util.tsx`

Reply:
oh sorry, i meant that the `Chain` type accepted in the query was a gQL definition and i didn't see an array of that type in `src/constants/chains.ts`.

`BACKEND_SUPPORTED_CHAINS` definitely works though

Reply:
actually, i don't think we can import product code into these .js scripts. i always get this error when i try to do it:

`SyntaxError: Cannot use import statement outside a module`

Reply:
you'll need to use require syntax, try: `const { BACKEND_SUPPORTED_CHAINS: chains } = require('../src/graphql/data/util.tsx')`

Reply:
that's what i did! i think these scripts aren't compatible with our current build setup

Reply:
okay I think we're stuck unless we can get this const in the sdk. Can you update the chain migration information to also include updating this constant?